edgexfoundry/app-service-configurable
# refactor!:Change configuration profile formats to YAML

BREAKING CHANGE: Configuration profiles now uses YAML format







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
1. start up edgex with no-secty and ds-virtual
2. `make build app-service-configurable`
3. run `./app-service-configurable -p {profile_name} -cp`
4. verify app service starts up with no error
5. stop app-service-configurable
6.   run `./app-service-configurable -p {profile_name} -cp -o`
7.  verify app service starts up with no error

## New Dependency Instructions (If applicable)
